### PR TITLE
Opgops 1706

### PR DESCRIPTION
**Pull request by tim-mcarthur-moj**

### CODE REVIEW

File: roles/ecs-cluster/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+- name: Create clsuters
```
Comment (carinadigital) [inline roles/ecs-cluster/tasks/main.yml]: typo

File: roles/ecs-cluster/tasks/destroy.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+- name: Ensure clusters are destroyed
+  ecs_cluster:
+    name: "{{ item }}-{{ target }}"
+    state: absent
+  register: cluster_data
```
Comment (carinadigital) [inline roles/ecs-cluster/tasks/destroy.yml]: Why is registering a variable needed in the destroy?

File: roles/ecs-cluster/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  include: create.yml
+  when: not destroy_stack|d(False)
```
Comment (carinadigital) [inline roles/ecs-cluster/tasks/main.yml:4]: d() short form of default() ?

Comment (tim-mcarthur-moj) [reply roles/ecs-cluster/tasks/main.yml:4]: yes.

File: roles/ecs-cluster/tasks/destroy.yml (hunk 2 above)
Comment (tim-mcarthur-moj) [reply roles/ecs-cluster/tasks/destroy.yml]: probably isn't

**Pull request merged**